model/src/w3profsmd_pdlib.F90

```diff
@@ -470,18 +364,25 @@ SUBROUTINE PDLIB_INIT(IMOD)
       ALLOCATE(IS0_pdlib(NSPEC))
       DO ISP=1, NSPEC
         IS0_pdlib(ISP) = ISP - 1
-      END DO
+1      END DO
```

Comment:
Is the 1 at the beginning of this line intentional?

Reply:
hmm, this must be some typo.